modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -566,199 +542,6 @@ function inope_policy_review_update_query(&$form_state) {
 }
 
 /**
- * Retrieve admin table from cache if possible.
```

Comment:
it comes up a little bit later in the exercises, so there will be a space to think about it then -- however, I think i want to illustrate that when you are proposing a different or new solution - you kind of do need to hit a point to point, if this were released as working functionality - which is the intent (with a few weeks or so going by with this deployed) and the system had any kind of load or volume  you can bet the fact that there is no caching would be noticed.